website_product_document_download_counter/models/product_document.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2025 Cetmix OÜ
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/LGPL).
+
+from odoo import _, fields, models
+from odoo.exceptions import UserError
+
+
+class ProductDocument(models.Model):
+    _inherit = "product.document"
+
+    download_count = fields.Integer(
+        default=0,
+        readonly=True,
+        copy=False,
+        help="Number of times this document has been downloaded",
+    )
+
+    count_download_enabled = fields.Boolean(
```

Comment:
I still think the prefix should be the same:

```suggestion
    download_count_enabled = fields.Boolean(
```